pkg/maps/lbmap/affinity.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
I wonder if `lbmap` is worth folding into `loadbalancer` codeowner or if it should stay in `bpf`.

Reply:
Yes, makes sense to add to `loadbalancer`. Shall I do it in this PR?